include/deal.II/gmsh/utilities.h

```diff
@@ -94,6 +104,60 @@ namespace Gmsh
     Triangulation<2, spacedim> &tria,
     const AdditionalParameters &prm = AdditionalParameters());
 #  endif
+
+#  ifdef DEAL_II_GMSH_WITH_API
+  /**
+   * Read a partitioned GMSH mesh file and create a fully distributed
+   * triangulation in deal.II.
+   *
+   * This function reads partitioned mesh files generated by GMSH's internal
+   * partitioner (usually metis) and constructs a fully distributed
+   * triangulation that can be used in parallel computations with MPI. The
+   * function automatically handles ghost cell identification, vertex
+   * coordinates, and cell connectivity across processor boundaries.
+   *
+   * In order to generate a set of partitioned mesh files that can be used with
+   * this function, you could use the `GMSH` command line tool with the
+   * following options:
+   * @code
+   * gmsh -part 4 -part_ghosts -part_split -save {input_file}
+   * @endcode
+   *
+   * This will generate a set of mesh files named like `{input_file}_1.msh`,
+   * `{input_file}_2.msh`, etc.
+   *
+   * In the command above, {input_file} can be any valid GMSH mesh input file.
```

Comment:
Oh, or maybe I misinterpret? Is it a gmsh input file, or an output file? If the latter, what happens if it's a file of the form `input.txt` -- does it create `input.txt_1.msh` etc?

Reply:
From GMSH perspective, it is an input file, that gmsh then processes. I.e., it can be a `.geo` file, a `.msh` file, or even a `.vtk`, `.inp`, `.ply`, etc. Any input file that gmsh knows how to handle. In my experiments. The output of the partitioner, however, seems to only support the `msh` format.

To recap:
```
gmsh -part 4 -part_ghosts -part_split -save some_grid.msh
```
will read `some_grid.msh`, and write `some_grid_1.msh`, `some_grid_2.msh`, `some_grid_3.msh`, `some_grid_4.msh`. 

But you could do 
```
gmsh -part 4 -part_ghosts -part_split -save -format msh some_grid.vtk
```
if you had a `.vtk` input grid, and this will generate the same 4 files above.

Reply:
I think that would be a useful example in the documentation of this function.